src/DynamoCore/Extensions/ExtensionManager.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Dynamo.Extensions
+{
+    public class ExtensionManager: IExtensionManager
```

Comment:
This will also need to implement `ILogSource` so that `ExtensionLoader` can also emit errors.